src/backend/marsha/core/views.py

```diff
@@ -372,6 +376,88 @@ def _get_app_data(self, cache_key, lti=None, resource_id=None):
 
         return app_data
 
+    def get_direct_access_from_liveregistration(
+        self, video_pk, liveregistration_pk, key
+    ):
```

Comment:
There is no cache at all. The JWT token can't be cached of course but all the app_data can.